spec/components/schemas.yaml

```diff
@@ -1072,6 +1072,8 @@ components:
           $ref: "#/components/schemas/datetime"
         end_datetime:
           $ref: "#/components/schemas/datetime"
```

Comment:
Is it meant to be on both versions of our API (v0.9 and v1)

Reply:
AFAIK the v0.9 spec should not change anymore. The new feature should only be in the v1 spec.
To keep things easier (and as it is not a breaking change), I decided to add the field to response of both versions. We could also check for v0.9 and remove the 'expires' field from the response again.